tests/vrf/test_vrf.py

```diff
@@ -933,7 +933,7 @@ def test_vrf_swss_warm_reboot(self, duthost, cfg_facts, partial_ptf_runner):
         assert wait_until(300, 20, check_interface_status, duthost, up_ports), \
                "All interfaces should be up!"
 
-    def test_vrf_system_warm_reboot(self, duthost, cfg_facts, partial_ptf_runner):
+    def test_vrf_system_warm_reboot(self, duthost, localhost, cfg_facts, partial_ptf_runner):
```

Comment:
This change is already in https://github.com/Azure/sonic-mgmt/pull/2338/files
If any of these two PR get merged first, the other will be conflicting. Can you resolve this?

Reply:
Fixed.